stories/dooboo-ui/Select.stories.tsx

```diff
@@ -0,0 +1,278 @@
+          <ThemeSwitch
+            value={disabled}
+            onValueChange={(): void => setDisabled(!disabled)}
+          />
+        </View>
+      </TogglerWrapper>
+      <CustomSelectContainer>
+        <Select
+          opened={isOpen}
+          loading={isLoading}
+          disabled={disabled}
+          showArrow={showArrow}
+          isDarkMode={darkTheme}
+          bordered={borderless}
+          activeOpacity={0.5}
+          placeholder={'New fancy select'}
+          value={selectedValue}
+          onSelect={(value): void => delaySelect(value)}
+          onOpen={(isOpen): void => toggleSelect(isOpen)}>
+          <Item value={'Item-1'}>{'Item-1'}</Item>
+          <Item value={'Item-2'}>{'Item-2'}</Item>
+          <Item value={'Item-3'}>{'Item-3'}</Item>
+        </Select>
+      </CustomSelectContainer>
+    </CustomContainer>
+  );
+};
+
+const Customized = (): React.ReactElement => {
+  const [listHeight, onChangeListHeight] = React.useState<string>('');
+  const [customPrefix, setCustomPrefix] = React.useState<string>(
+    'https://user-images.githubusercontent.com/50701501/88152214-ae263680-cc3e-11ea-9a72-062e0208ee79.png',
+  );
+  const [customSuffix, setCustomSuffix] = React.useState<string>(
+    'https://user-images.githubusercontent.com/50701501/88151403-9dc18c00-cc3d-11ea-95c0-447162f8465e.png',
+  );
+
+  // [Select] states
+  const [selectedValue, onSelectValue] = React.useState<
+    string | number
+  >();
+  const [isOpen, toggleSelect] = React.useState<
+    boolean
+  >(false);
+  const [isLoading, setLoading] = React.useState<
+    boolean
+  >(false);
+
+  const delaySelect = (selectedItem) : void => {
+    setLoading(true);
+    setTimeout(() => {
+      setLoading(false);
+      onSelectValue(selectedItem);
+    }, 700);
+  };
+
+  return (
+    <CustomContainer>
+      <TogglerWrapper>
+        <View style={{ alignSelf: 'center' }}>
+          <CustomText>{'Fixed List height'}</CustomText>
+          <CustomTextInput
+            placeholder={'Set or Auto-sized'}
+            value={listHeight}
+            onChangeText={(text): void => onChangeListHeight(text)}
+          />
+          <CustomText>{'Custom Prefix icon'}</CustomText>
+          <CustomTextInput
+            style={{ width: 300 }}
+            value={customPrefix}
+            onChangeText={(text): void => setCustomPrefix(text)}
+          />
+          <CustomText>{'Custom Suffix icon'}</CustomText>
+          <CustomTextInput
+            style={{ width: 300 }}
+            value={customSuffix}
+            onChangeText={(text): void => setCustomSuffix(text)}
+          />
+        </View>
+      </TogglerWrapper>
+      <CustomSelectContainer style={{ height: 60, width: 200 }}>
+        <Select
+          opened={isOpen}
+          loading={isLoading}
+          disabled={false}
+          showArrow={true}
+          activeOpacity={0.9}
+          listHeight={Number(listHeight)}
```

Comment:
`listHeight` from `TextInput` value to number and then setting it to `listHeight` value? I hope I can understand your intention here.